migrations/2024-07-01-163929_user_password_encrypted_optional/down.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE local_user
+    ALTER COLUMN password_encrypted SET NOT NULL;
```

Comment:
This can fail if oauth accounts were already created. Not sure what to do about that.